omod/src/main/java/org/openmrs/module/queue/web/resources/QueueEntryCountSubResource.java

```diff
@@ -37,48 +38,48 @@
  * For Service&v=custom:(count) -returns only the count.
  */
 @SuppressWarnings("unused")
-@SubResource(parent = QueueResource.class, path = "count", supportedClass = QueueCount.class, supportedOpenmrsVersions = {
+@SubResource(parent = QueueResource.class, path = "count", supportedClass = QueueEntry.class, supportedOpenmrsVersions = {
         "2.0 - 2.*" })
-public class QueueCountSubResource extends DelegatingSubResource<QueueCount, Queue, QueueResource> {
+public class QueueEntryCountSubResource extends DelegatingSubResource<QueueEntry, Queue, QueueResource> {
 	
 	@Override
-	public Queue getParent(QueueCount queueCount) {
-		return queueCount.getQueue();
+	public Queue getParent(QueueEntry queueEntryCount) {
+		return queueEntryCount.getQueue();
 	}
 	
 	@Override
-	public void setParent(QueueCount queueCount, Queue queue) {
-		queueCount.setQueue(queue);
+	public void setParent(QueueEntry queueEntryCount, Queue queue) {
+		queueEntryCount.setQueue(queue);
 	}
 	
 	@Override
 	public PageableResult doGetAll(Queue queue, RequestContext requestContext) throws ResponseException {
```

Comment:
Hadn't really clocked this before... It seems like this is a weird thing to do in `doGetAll()`, but I suppose if it works....